src/main/java/com/core/service/mapper/UserMapperImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.core.service.mapper;
+
+import com.core.model.User;
+import com.core.model.dto.UserRequestDto;
+import com.core.model.dto.UserResponseDto;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import lombok.AllArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Service;
+
+@AllArgsConstructor
+@Service
+public class UserMapperImpl
```

Comment:
Do we need to create mapper and dto in first task?

Reply:
Nope, but I did